climpred/classes.py

```diff
@@ -91,6 +98,56 @@ def __init__(self, xobj):
     def __repr__(self):
         return _display_metadata(self)
 
+    def smooth(self, smooth_dict='goddard2013'):
+        """Smooth all entries of PredictionEnsemble in the same manner to be
+        able to still calculate prediction skill afterwards.
+
+        Args:
+          xobj (xarray object):
+            decadal prediction ensemble output.
+
+        Attributes:
+            smooth_dict (dict or str): Dictionary to specify the dims to
+                smooth combaticle with `spatial_smoothing_xesmf`,
+                `temporal_smoothing` or `spatial_smoothing_xrcoarsen`.
+                Shortcut for Goddard et al. 2013 recommendations:
+                'goddard2013'
+        """
+        # get proper smoothing function based on smooth args
+        if isinstance(smooth_dict, str):
+            if smooth_dict == 'goddard2013':
+                smooth_fct = smooth_goddard_2013
+                smooth_dict = {'lead': 4}  # default
```

Comment:
Similarly, need to figure out if this holds for non-annual averages. Maybe just have the user manually put this in.

Reply:
Once temporal PR is ready and we know the requirements what needs to be coded there, this can be implemented